frontend/talentsearch/src/js/components/experienceAndSkills/ExperienceAndSkills.tsx

```diff
@@ -250,11 +253,7 @@ export const ExperienceAndSkills: React.FunctionComponent<
         </div>
       )}
       {!hasExperiences ? (
-        <div
-          data-h2-radius="base(s)"
-          data-h2-background-color="base(light.dt-gray)"
-          data-h2-padding="base(x1)"
-        >
+        <Well>
```

Comment:
this renders a bit funny looking, a real nitpicky thing
![image](https://user-images.githubusercontent.com/40485260/192633175-6ce5de67-6a05-43cd-b22f-089fa64e3f6f.png)

Reply:
looking at 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f045b99dfa414efc2b2f18556ab52a2d1574c58f/frontend/admin/src/js/components/user/GeneralInformationTab/GeneralInformationTab.tsx#L87-L95

it might be due to one being nested in `<p></p>` and the other not

Reply:
I'm a little confused, the code you linked to does not display the text I am seeing in the screenshot. Where are you seeing that?

Reply:
oh that was me comparing the rendering of this line with notes comment bit from above, the point I was trying to make was the `<p> `was present in one but not the other and was probably the cause of the discrepancy in spacing
sorry

Reply:
I'm referring to the `ExperienceAndSkills.tsx`. I'm not seeing the what you see in the screenshot and it is coded to say:

"There are no experiences on your profile yet. You can add some using the preceding buttons."

Does your screenshot perhaps appear elsewhere in the codebase?

Reply:
okay, I think this is the correct one for the screenshot in this chain, I had the wrong file
if I still didn't get the right file I am going to delete this review out of embarrassment 😵 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/f045b99dfa414efc2b2f18556ab52a2d1574c58f/frontend/talentsearch/src/js/components/skills/SkillsInDetail/SkillsInDetail.tsx#L181-L192

Reply:
ohh, okay. Yeah, that was update 😛